astroquery/esa_hubble/core.py

```diff
@@ -0,0 +1,422 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Javier Duran
+@contact: javier.duran@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 13 Ago. 2018
+
+
+"""
+from astroquery.utils import commons
+from astropy import units
+from astropy.units import Quantity
+from astroquery.utils.tap.core import TapPlus
+from astroquery.utils.tap.model import modelutils
+from astroquery.query import BaseQuery
+from astropy.table import Table
+from six import BytesIO
+
+from . import conf
+from astropy import log
+
+
+__all__ = ['EsaHubble', 'EsaHubbleClass']
+
+
+class ESAHubbleHandler(BaseQuery):
+
+    def __init__(self):
+        super(ESAHubbleHandler, self).__init__()
+
+    def get_file(self, filename, response, verbose=False):
+        with open(filename, 'wb') as fh:
+            fh.write(response.content)
+
+        if "/" not in filename:
```

Comment:
would this work on windows?

Reply:
This change was introduced by @keflavich if I recall correctly. Should I go back to the previous way?

Reply:
I don't recall doing this, but perhaps I did.  In any case, it's not very important functionality, as it's just governing what log message is reported.

Reply:
Maybe @bsipocz is referring here only to the "/" part. I can modify it if you prefer by os.pathsep